File: contest165/b.py


def main():
	import sys 
	input=sys.stdin.readline 

	money=100
	x=int(input())

	for i in range(1,4000):
		money *=0.01 
		money=int(money)

		if money >=x:
			print(i)
			exit()

if __name__=='__main__':
	main()
	

File: contest165/b.cpp
#include <iostream>
#include <cstdlib>
using namespace std;



int main(){
	int x ;
	cin >>x ;

	int money=100;

	for(int i=1;i<4000;i++){
		money=money*0.01;
		money =int(money);

		if(money>=x){
			cout << i <<"\n";
			_Exit(0);
		}

	}

	return 0;
}